spec/mailers/user_mailer_spec.rb

```diff
@@ -834,6 +841,31 @@ def expect_email_body_to_have_help_and_contact_links
             t('in_person_proofing.process.state_id.heading'),
           )
       end
+
+      context 'For Enhanced In-Person Proofing (Enhanced IPP)' do
+        let(:enrollment) { enhanced_ipp_enrollment }
+        it 'renders content that is applicable to Enhanced In-Person Proofing (Enhanced IPP)' do
+          aggregate_failures do
+            [
+              t('in_person_proofing.headings.barcode_what_to_bring'),
+              t('in_person_proofing.body.barcode.what_to_bring'),
+              t('in_person_proofing.process.eipp_bring_id.heading'),
+              t('in_person_proofing.process.eipp_bring_id.info'),
+              t('in_person_proofing.process.eipp_state_id_passport.heading'),
+              t('in_person_proofing.process.eipp_state_id_passport.info'),
+              t('in_person_proofing.process.eipp_state_id_military_id.heading'),
+              t('in_person_proofing.process.eipp_state_id_military_id.info'),
+              t('in_person_proofing.process.eipp_state_id_supporting_docs.heading'),
+              t('in_person_proofing.process.eipp_state_id_supporting_docs.info'),
+
+            ].each do |copy|
+              Array(copy).each do |part|
+                expect(mail.html_part.body).to have_content(part)
+              end
+            end
+          end
+        end
+      end
```

Comment:
Non-blocking: What do you think about adding a For In-Person Proofing (IPP) context block to test for the items that EIPP does not have? It looks like we just check the heading of the IPP version for the reminder.

Reply:
Done 👍🏻